.github/workflows/pytest.yml

```diff
@@ -8,41 +8,13 @@ on:
 
 jobs:
   pyTestCov:
-    runs-on: ubuntu-latest
```

Comment:
What happened to this? Included in python@v2?

Reply:
By this, I mean lines 11-38

Reply:
@vkbo

Reply:
I just merged the two jobs into one.

Reply:
Oh, now I see. Thanks

Reply:
For context: They were previously two job definitions with the first having coverage, and the second being a job matrix without coverage. But I found out that codecov just merges multiple coverage reports for the same commit, so there was no point not running coverage on all jobs.